hw/hw11/mailroom.py

```diff
@@ -0,0 +1,90 @@
+donors = {
+    'Homer': [25, 10, 20],
+    'Marge': [100, 50, 75],
+    'Bart': [5, 1, 2],
+    'Lisa': [5, 10, 12, 6]}
+
```

Comment:
Best to end a data collection with a single closed `}` on it's own line here instead of line 5. It is easier to read.